modulemd/modulemd-modulemetadata.c

```diff
@@ -121,6 +121,7 @@ modulemd_modulemetadata_set_name (ModulemdModuleMetadata *self,
                                   const gchar            *name)
 {
   g_return_if_fail (MODULEMD_IS_MODULEMETADATA (self));
+  g_return_if_fail (name);
```

Comment:
These were explicitly left out because it's the only way to cause the value to be cleared entirely. That was a desired behavior. I understand the desire to validate input, but this was intentional. I will add this to the gtk-doc so it is clear.

Patch is rejected.